### PR TITLE
move [Test]LambdaMARTModel from ltr.(feature|ranking) to ltr.model

### PR DESCRIPTION
**Pull request by cpoerschke**

### CODE REVIEW

File: solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LambdaMARTModel.java [hunk 1]
```diff
@@ -23,9 +23,8 @@
 
 import org.apache.lucene.index.LeafReaderContext;
 import org.apache.lucene.search.Explanation;
-import org.apache.solr.ltr.model.LTRScoringModel;
-import org.apache.solr.ltr.model.ModelException;
 import org.apache.solr.ltr.norm.Normalizer;
+import org.apache.solr.ltr.ranking.Feature;
```
Comment (cpoerschke) [inline solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LambdaMARTModel.java]: Moving Feature.java from ltr.ranking to ltr.feature would be a next separate pull request.

**Comment by diegoceccarelli** [comment]
LGTM
